web/src/main/java/co/mobileaction/example/web/service/PostService.java

```diff
@@ -42,4 +42,10 @@ public void deletePost(Long postId)
     {
         postRepository.deleteById(postId);
     }
+
+    @Override
+    public void deleteAllPostsOfUser(Long userId){

```

Comment:
Well approach, but this approach uses db two times. One for finding related posts and second one is deleting the related posts. 
There is a more effecient way to delete all posts of the related user. Please reconsider this approach 👍

Reply:
You can use @Query annotation for solution 👍